resources/core/charts/k8s-dashboard-proxy/templates/virtual-service.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
```

Comment:
We don't need a new Gateway here. The dashboard-proxy uses the "global" one, see"
```
- {{ .Values.global.istio.gateway.name }}
```

Reply:
yes, removed already